test/updaters/package-json.ts

```diff
@@ -36,6 +36,25 @@ describe('PackageJson', () => {
     });
 
     it('updates dependency versions', async () => {
+      const oldContent = readFileSync(
+        resolve(fixturesPath, './package-with-dependencies.json'),
+        'utf8'
+      );
+      const versionsMap: VersionsMap = new Map();
+      versionsMap.set('camelcase', Version.parse('6.0.0'));
+      versionsMap.set('chai', Version.parse('4.2.1'));
+      versionsMap.set('foo', Version.parse('0.1.0'));
+      versionsMap.set('bar', Version.parse('2.3.4'));
+      const packageJson = new PackageJson({
+        version: Version.parse('14.0.0'),
+        versionsMap,
+        updatePeerDependencies: true,
```

Comment:
This test implicitly dependended on the updater always updating the peer dependencies. So it was updated and produces the same output.